src/browser/provider/built-in/chrome/create-temp-user-dir.js

```diff
@@ -0,0 +1,45 @@
+import path from 'path';
```

Comment:
I guess we should rename `create-temp-user-dir.js` to `create-profile.js` or `create-temp-profile.js` because this name better describes the function behaior.

The function do:
* create a temporary directory
* create 'Preferences' file
* create 'First Run' file
* write them to the temporary directory